jira.py

```diff
@@ -253,7 +335,7 @@ def download_projects(project_ids: List[str]):
 def save_cache():
     # store all the tickets
     print("Total number of tickets: {0}".format(len(tickets_json)))
-    save_json(tickets_title, tickets_json)
+    save_json(tickets_title, tickets_json, True)
```

Comment:
Adding `pretty_print == True` here would mean that the cache size will blow up for users, who download a lot of Jira issues. For example, for my use case, the non-pretty-printed cache was over three GBs. It mustn't be pretty printed both for space and performance reasons.

```suggestion
    save_json(tickets_title, tickets_json)
```